astropy/cosmology/io/tests/test_ecsv.py

```diff
@@ -127,6 +127,29 @@ def test_readwrite_ecsv_instance(
         got = read(fp)
         assert got == cosmo
 
+    def test_readwrite_ecsv_rename(
+        self, cosmo_cls, cosmo, read, write, tmp_path, add_cu
+    ):
+        """Test rename argument to read/write."""
+        fp = tmp_path / "test_readwrite_ecsv_rename.ecsv"
+        rename = {"name": "cosmo_name"}
+
+        write(fp, format="ascii.ecsv", rename=rename)
+
+        tbl = QTable.read(fp, format="ascii.ecsv")
+
+        assert "name" not in tbl.colnames
+        assert "cosmo_name" in tbl.colnames
+
+        # Errors if reading
+        with pytest.raises(TypeError, match="there are unused parameters"):
```

Comment:
I'd like the `match` value to be even more detailed. Looking at this test it is not at all clear to me where the unused parameters are.

Reply:
Done!